src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt

```diff
@@ -66,26 +66,8 @@ abstract class Address(
   @Column(name = "POSTAL_CODE")
   open val postalCode: String? = null,
 
-) {
```

Comment:
city, county, country moved up to constructor - PR diff just looks like the opposite has happened